README.org

```diff
@@ -1,10 +1,21 @@
-#+title: Inventory users that have sudo access
+The =sudo= program allows a user to execute a command with elevated privileges.
+It is a good idea to limit and monitor who has access to do this.
+This module will check which users on a system can use sudo (=sudoers=) and report that as part of CFEngine Enterprise Inventory.
+In Mission Portal, the CFEngine Enterprise UI, you can get reports of all users with sudo access across all machines in your infrastructure:
 
-* Synopsis
+[[https://raw.githubusercontent.com/nickanderson/cfengine-inventory-sudoers/main/inventory-screenshot.png][[[https://raw.githubusercontent.com/nickanderson/cfengine-inventory-sudoers/main/inventory-screenshot.png]]]]
```

Comment:
This is horrible. The image won't load in org-mode.

Reply:
I don't have a better way currently.

Reply:
Also, looks like some extra `[]` in there.

Should be `[[http://target][anchor text]]`

Reply:
@nickanderson it's supposed to be an image that is a link to itself.

Reply:
HATE